_data/people/msneubauer.yml

```
@@ -49,3 +49,38 @@ presentations:
     meetingurl: https://www.jlab.org/indico/event/357/
     location: Jefferson Laboratory
     focus-area: as, ia
+  - title: Future Blueprint Topics: Year 3 Plans, 4 and 5 Year Thoughts
```

@msneubauer This is going to need to be quoted given the colon
```suggestion
  - title: "Future Blueprint Topics: Year 3 Plans, 4 and 5 Year Thoughts"
```